### PR TITLE
Fix/13 function naming convention

### LINKED ISSUE #10: incorrect access to config_plugins tables
please use get_config() api and not direct access to config_plugins tables - the config data is cached within MUC.
https://github.com/eLearning-BS23/quizaccess_invigilator/blob/master/lib.php#L63)

### LINKED ISSUE #13: Function names must follow rules
"get_invigilator_settings" is incorrect.

https://github.com/eLearning-BS23/quizaccess_invigilator/blob/master/lib.php#L60

more info see:
https://docs.moodle.org/dev/Coding_style#Functions_and_Methods

### PR DESCRIPTION
**Pull request by shadman-ahmed-bs23**
PR for issue #10 and #13

### CODE REVIEW

File: rule.php [hunk 1]
```diff
@@ -96,8 +96,8 @@ public function make_modal_content() {
     public function add_preflight_check_form_fields(mod_quiz_preflight_check_form $quizform, MoodleQuickForm $mform, $attemptid) {
         global $PAGE;
         $coursedata = $this->get_courseid_cmid_from_preflight_form();
-        $screenshotdelay = get_invigilator_settings('screenshotdelay');
-        $screenshotwidth = get_invigilator_settings('screenshotwidth');
+        $screenshotdelay = quizaccess_invigilator_get_invigilator_settings('screenshotdelay');
```
Comment (sahedbs23) [inline rule.php]: @shadman-ahmed-bs23  can we use get config method directly without introduce another function get_invigilator_settings

Ex. get_config('quizaccess_invigilator', 'screenshotdelay')

File: rule.php [hunk 2]
```diff
@@ -96,8 +96,8 @@ public function make_modal_content() {
...
+        $screenshotwidth = quizaccess_invigilator_get_invigilator_settings('screenshotwidth');
```
Comment (sahedbs23) [inline rule.php]: See Previous review.

File: rule.php [hunk 3]
```diff
@@ -301,8 +301,8 @@ public function setup_attempt_page($page) {
         global $DB, $COURSE, $USER;
         if ($cmid) {
             // Get Screenshot Delay and Image Width.
-            $screenshotdelay = get_invigilator_settings('screenshotdelay');
-            $screenshotwidth = get_invigilator_settings('screenshotwidth');
+            $screenshotdelay = quizaccess_invigilator_get_invigilator_settings('screenshotdelay');
```
Comment (sahedbs23) [inline rule.php]: See Previous review.

File: rule.php [hunk 4]
```diff
@@ -301,8 +301,8 @@ public function setup_attempt_page($page) {
...
+            $screenshotwidth = quizaccess_invigilator_get_invigilator_settings('screenshotwidth');
```
Comment (sahedbs23) [inline rule.php]: See Previous review.

**Review by sahedbs23 — CHANGES_REQUESTED** [review]

File: rule.php (hunk 1 above)
Comment (shadman-ahmed-bs23) [reply rule.php]: @sahedbs23 bhai, replaced the function with get_config(). Please check

**Pull request merged**